supports-color/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": false,
```

Comment:
Does https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/README.md#create-a-new-package needs to be updated to reflect that?